StudCity.Frontend/src/App.js

```diff
@@ -5,8 +5,9 @@ import {
 } from "react-router-dom";
 
 import Main from "./pages/Main";
-import Authenticate from "./pages/Authenticate";
-import Registration from "./pages/Registration";
+import Authenticate from "./pages/authenticate";
+import Registration from "./pages/registration";
+import VerifyEmail from "./pages/verify-email/VerifyEmail";
```

Comment:
please use naming index.jsx file instead of VerifyEmail.jsx so that the import route look like in lines 8-9